update vmbackup/vmrestore README usage (#794)
VictoriaMetrics/VictoriaMetrics: app/vmrestore/README.md

### Usage

> VictoriaMetrics must be stopped during restore process.

Reviewer: it was correct before

```
vmrestore -src=gcs://<bucket>/<path/to/backup> -storageDataPath=<local/path/to/restore>

```

* `<bucket>` is [GCS bucket](https://cloud.google.com/storage/docs/creating-buckets) name.

> * `<path/to/backup>` is the path to back up made with [vmbackup](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmbackup/README.md) on GCS bucket.

Reviewer: it was correct before - backup is noun here

* `<local/path/to/restore>` is the path to folder where data will be restored. This folder must be passed

>   to VictoriaMetrics in `-storageDataPath` command-line flag after restore process is complete.

Reviewer: it was correct before

The original `-storageDataPath` directory may contain old files. They will be substituted by the files from backup,
i.e. the end result would be similar to [rsync --delete](https://askubuntu.com/questions/476041/how-do-i-make-rsync-delete-files-that-have-been-deleted-from-the-source-folder).

### Troubleshooting

* If `vmrestore` eats all the network bandwidth, then set `-maxBytesPerSecond` to the desired value.

> * If `vmrestore` has been interrupted due to temporary error, then just restart it with the same args. It will resume restore process.

Reviewer: it was correct before

Run `make package-vmrestore`. It builds `victoriametrics/vmrestore:<PKG_TAG>` docker image locally.
`<PKG_TAG>` is auto-generated image tag, which depends on source code in the repository.

> The `<PKG_TAG>` maybe manually set via `PKG_TAG=foobar make package-vmrestore`.

Reviewer 2: ```suggestion
The `<PKG_TAG>` may be manually set via `PKG_TAG=foobar make package-vmrestore`.
```

> By default, the image builds on top of [alpine](https://hub.docker.com/_/alpine) image. It is possible to build the package on top of any other base image

Reviewer 2: ```suggestion
The base docker image is [alpine](https://hub.docker.com/_/alpine) but it is possible to use any other base image
```
